wao/wao_strategy_controller.py

```diff
@@ -1,8 +1,13 @@
-from wao.brain_util import perform_execute_buy, perform_execute_sell, write_to_backtest_table, clear_cumulative_value, create_initial_account_balance_binance_file, is_romeo_alive, remove_from_pool
+# Revert
+import sys
+sys.path.append("wao/")
```

Comment:
without line 3 code, i was having this error 'no module named wao'.

Reply:
it's because it was called from wao/controller_test, in production it is never called from wao/, please revert the whole file

Reply:
this is test code, do not push test code

Reply:
resolved.